demo/addons/godot-openxr/plugin.cfg

```diff
@@ -0,0 +1,7 @@
+[plugin]
+
```

Comment:
Note, adding the plugin section allows us to add editor based stuff, disabling the plugin here only disables the editor enhancements not the actual plugin. We could have some more fun with this in due time.